webClient/src/app/core/menu-bar/menu-bar.config.ts

```diff
@@ -39,57 +39,43 @@ export const LANGUAGE_MENUS = {
   'jcl': [
     {
       name: 'Submit',
-      isDisabledString: `
-      const plugin = ZoweZLUX.pluginManager.getPlugin('org.zowe.explorer-jes');
-      const file = context.controller.fetchActiveFile();
-    if (!plugin || !file || (ZoweZLUX.uriBroker.serverRootUri('') == '/')) {
-        return true;
-      }
-      return false;
-      `,
       action: {
-        /*
-          TODO z/osmf has a jobs api, so this makes use of it for now. 
-          But, we don't import that service into the editor, because it would make a hard requirement instead of an optional one.
-          May want to have such metadata in plugindef, or perhaps this is an interface & capability to be searched up
-        */
         functionString:`
         const file = context.controller.fetchActiveFile();
         if (file) {
-          let content = context.editor.model.getValue();
+          let content = context.controller.fetchActiveFile().model.contents + '\\n';
           if (content && content.length > 0) {
             content = content.replace(/\\n/g,'\\\\n');
-            const uri = '/api/v1/jobs/string';
-            const stringJsonBody = '{ "jcl": "'+content+'"}';  
-            fetch(uri, {method: 'POST', body: stringJsonBody,
+            const uri = '/jes';
```

Comment:
Instead of `/jes` you want `ZoweZLUX.uriBroker.agentRootUri('jes')` because it knows whether or not to go through APIML
https://github.com/zowe/zlux-app-manager/blob/v1.x/staging/bootstrap/src/uri/mvd-uri.ts#L149

Reply:
Done